Kiosk/Bid Fulfillment/BidCheckingNetworkModel.swift

```diff
@@ -33,48 +54,35 @@ class BidCheckingNetworkModel: NSObject {
         // If polling fails, we can still show bid confirmation. Do not error.
         } .catchTo( RACSignal.empty() )
     }
-
-    private func pollForUpdatedSaleArtwork() -> RACSignal {
-
-        let beginningBidCents = fulfillmentController.bidDetails.saleArtwork?.saleHighestBid?.amountCents ?? 0
-
-        let updatedSaleArtworkSignal = getUpdatedSaleArtwork().flattenMap { [weak self] (saleObject) -> RACStream! in
+    
+    private func pollForUpdatedBidderPosition(bidderPositionId: String) -> RACSignal {
+        let updatedBidderPosition = getUpdatedBidderPosition(bidderPositionId).flattenMap { [weak self] bidderPositionObject in
             self?.pollRequests++
-
+            
             logger.log("Polling \(self?.pollRequests) of \(self?.maxPollRequests) for updated sale artwork")
-
-            let saleArtwork = saleObject as? SaleArtwork
-            let updatedBidCents = saleArtwork?.saleHighestBid?.amountCents ?? 0
-
+            
             // TODO: handle the case where the user was already the highest bidder
```

Comment:
There are scripts like https://github.com/artsy/gravity/blob/master/lib/auction_create.rb, but I haven't tried doing it myself yet. My understanding is that we can invoke these from the staging console. Probably can also create an auction in Ohm.